docs/sources/experiments/cohort-labels.md

```diff
@@ -55,24 +55,27 @@ Triage expects the cohort query passed to it to return a unique list of `entity_
 
 #### Inspections Cohort Config
 
-The way this looks in an Experiment configuration YAML is as follows:
+We recommend storing your cohort queries as files in their own directory. This allows you to more easily reuse and update cohorts in multiple experiments. The way this looks in an Experiment configuration YAML is as follows:
```

Comment:
I think focusing on storing the queries in files here makes sense, but should we at least still mention the option of specifying them inline with a `query` key here as well?